cmd/errtrace/main.go

```diff
@@ -721,6 +730,14 @@ func (t *walker) isErrtraceWrap(expr ast.Expr) bool {
 		sel.Sel.Name == "Errorf"
 }
 
+// optout reports whether the line at the given position
+// is opted out of tracing.
+func (t *walker) optout(pos token.Pos) bool {
+	line := t.fset.Position(pos).Line
+	_, ok := t.optouts[line]
+	return ok
```

Comment:
optional: should we try to detect optouts that have no impact? we could `delete(..)` here if we find a match, and any optouts that remain would indicate invalid optouts?

Reply:
Makes sense. Can address in a follow up. Creating an issue.
Deleting won't do, though.

    return a, err1, err2 //errtrace:skip

This will hit the opt-out check twice.
So we'll have to maintain a second list of opt-outs that have been used, then tally them after the file has been processed.